crates/fuel-streams-core/src/nats/conn_stores.rs

```diff
@@ -0,0 +1,30 @@
+use super::store::{Storable, Store};
+use crate::{
+    nats::{NatsClient, NatsError},
+    types::{Block, Transaction},
+};
+
+#[derive(Clone)]
+pub struct ConnStores {
+    pub blocks: Store<Block>,
+    pub transactions: Store<Transaction>,
+}
```

Comment:
This public API from `fuel-streams-core` will allow streaming blocks and transactions. why call it it `Stores`?